build.gradle

```diff
@@ -1,7 +1,23 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    ext.buildConfig = [
+        'compileSdk': 30,
+        'buildToolsVersion': "30.0.2",
```

Comment:
isnt the build tools version picked automatically by AGP?

Reply:
Yep